plugp100/responses/device_state.py

```diff
@@ -81,28 +81,75 @@ def try_from_json(kwargs: dict[str, Any]) -> Either['LedStripDeviceState', Excep
 @dataclass
 class DeviceInfo:
     device_id: str
+    hardware_id: str
+    oem_id: str
+    timezone: str
     firmware_version: str
     hardware_version: str
+    ip: str
     mac: str
     nickname: str
     model: str
     type: str
     overheated: bool
+
+    # wifi
+    ssid: str
     signal_level: int
     rssi: int
+
+    # location data
+    latitude: int
+    longitude: int
+    timezone: str
+    time_difference: int
+    language: str
+
+    # device state
+    power_protection_status: str
+    device_on: bool
+    on_time: int
+
+    # autooff
+    auto_off: bool
+    auto_off_time_remaining: int
+
     is_hardware_v2: bool = property(lambda self: self.hardware_version == "2.0")
 
+
+
     def __init__(self, **kwargs):
         self.device_id = kwargs['device_id']
+        self.hardware_id = kwargs['hw_id']
+        self.oem_id = kwargs['oem_id']
         self.firmware_version = kwargs["fw_ver"]
         self.hardware_version = kwargs["hw_ver"]
         self.mac = kwargs["mac"]
         self.nickname = base64.b64decode(kwargs["nickname"]).decode("UTF-8")
         self.model = kwargs["model"]
         self.type = kwargs["type"]
         self.overheated = kwargs.get("overheated", False)
+        self.power_protection_status = kwargs["power_protection_status"]
+        self.ip = kwargs['ip']
```

Comment:
There is a mixture of double and single quotes throughout the code base. @petretiandrea, have you considered using black to enforce the wanted code style?

Reply:
Yes, I will add it